docs/conf.py

```diff
@@ -19,6 +19,11 @@
 
 from datetime import datetime
 
+# append the ddtrace path to syspath
+# this is required when building the docs manually
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath(".."))
```

Comment:
We had this line before but it was removed in #1633 since it breaks readthedocs. It does seem to be required for manual usage so it's probably worth keeping it here and commented.